src/ui/common/Sidebar/SidebarContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import SidebarComponent from "./SidebarComponent"
+import { useAppDispatch, useAppSelector } from "../../../app/hooks"
+import {RootState} from "../../../app/store";
+import { toggleStatus } from "../../../features/opener/sidebarSlice";
+import { useNavigate } from 'react-router-dom';
+import { useTheme } from '@mui/material/styles';
+
+const SidebarContainer = () => 
+{
+   const isOpened = useAppSelector((state: RootState) => state.sidebar.open)
+   const theme = useTheme();
+   const dispatch = useAppDispatch();
+   const navigate = useNavigate();
+   const handleSidebarOpen = () => 
+   {
+      dispatch(toggleStatus(!isOpened));
+   }
+   return (
+        <SidebarComponent isOpened = {isOpened} handleSidebarOpen = {handleSidebarOpen} theme={theme} navigate={navigate}/>
```

Comment:
formatting Use the same number of space

Reply:
Formatting has been improved using Prettier